integration/common.go

```diff
@@ -124,6 +128,10 @@ func CreateVerifiableCredential(credentialInput credInputParams, revocable bool)
 	if revocable {
 		fileName = "credential-revocable-input.json"
 	}
+	if suspendable {
+		fileName = "credential-suspendable-input.json"
```

Comment:
Maybe worth putting them in the same `json` and using the templating mechanism?

Reply:
Added 👍 and removed these .json files